influence_benchmark/experiments/hyperparameter_sweep.py

```diff
@@ -52,7 +52,7 @@ def train_loop(config=None):
             accelerate_config_path=accelerate_config_path,
             sft_script_path=sft_script_path,
             model_name=model_name,
-            num_gen_trajectories_per_state=config.num_gen_trajectories_per_state,
+            num_gen_trajectories_per_state=6,
```

Comment:
kind of depends which parameters we want to vary each run but we might not want to push version with 6 hardcoded

Reply:
I guess we're doing this?